### DISCUSSION TITLE
[question] USB power switch to Battery

### DISCUSSION THREAD

**Discussion by bushvin** [zwave-js/zwave-js-ui#914] [Q&A]
I have the AEON Labs Multisensor 6 device.
It can be connected to USB for power, or have internal batteries.

When pairing them to a controller they advise powering over USB, and then change to battery once paired.

I have noticed when adding them using USB differs from adding them on Battery.
A device paired with USB and then switched to Battery' status is `Dead` upon restart of the controller. It's status is reverted to `Alive` once it reports (after the configured sleep time, or by manually waking the device up).
A device paired using batteries' status is `Asleep`

Is this a known issue?

**Comment by robertsLando** [comment]
I think this is normal but for sure let's see what @AlCalzone says about this

**Reply by AlCalzone** [reply]
On inclusion, the devices tell the controller if they are always listening (permanently powered) or not. Since this information is only stored in the controller's memory on inclusion, it never learns that the device isn't actually powered permanently. 

By including with USB and then using the device on battery power, you essentially lie to the controller. The controller thinks it can use the device to route packets to otherwise unreachable nodes and (even worse!) tell those nodes to use the 99% unavailable multisensor for routing. zwave-js just uses the controller's information.

TLDR: That is terrible advice - I don't know why they print it in the manual. Don't do it!

**Reply by bushvin** [reply]
That does make sense, @AlCalzone .

It is indeed strange they would suggest such a thing. Thanks for the clarification!

Just for reference, here's the [guide](https://help.aeotec.com/support/solutions/articles/6000057073-multisensor-6-user-guide-) in question:

*For the purposes of installation and setup, even if you intend to power your sensor with batteries, we recommend using the provided USB cable for setup.*

Oh well, re-joining them all with batteries then!
